web/index.php

```diff
@@ -45,6 +45,7 @@ function () {
             ->with(new RequestMatcher('^/contexts/.*', null, 'GET'))
             ->with(new RequestMatcher('^/(event|place|label)/' . $app['id_pattern'] . '$', null, 'GET'))
             ->with(new RequestMatcher('^/(events|places)/' . $app['id_pattern'] . '$', null, 'GET'))
+            ->with(new RequestMatcher('^/(events|places)/' . $app['id_pattern'] . '/calSum+$', null, 'GET'))
```

Comment:
The + at the end of the path should be dropped, now you can use calSum or calSumm or calSummmmmmm and these would still match :)